src/oscar/templates/oscar/dashboard/catalogue/product_update.html

```diff
@@ -81,6 +81,7 @@ <h3>{% trans "Sections" %}</h3>
                                 {% if not parent %}
                                     <li><a href="#product_recommended" data-toggle="tab">{% trans 'Upselling' %}</a></li>
                                 {% endif %}
+                                <li><a href="#seo" data-toggle="tab">{% trans 'Search Enging Optimization' %}</a></li>
```

Comment:
Spelling - `Enging`

Reply:
Also please use the British english spelling for Optimisation everywhere, as en-gb is the default language for Oscar.